### DISCUSSION TITLE
Reset admin password?

### DISCUSSION THREAD

**Discussion by CharlesBeatty** [Budibase/budibase#2142] [Help] [answered]
I just built Budibase in a docker cluster. And then I must have fat fingered the administrator password. Now I can't log in. Is there any way to reset the admin password on the server or do I need to reinstall? I had not configured SMTP yet so the email reset does not work.

**Comment by shogunpurple** [comment, marked as answer]
@CharlesBeatty thanks for raising. 

So because SMTP has not been set up, there's not a nice way to do this. The easiest way if possible is to just clear all the containers and start again. 

The alternative is to delete your admin user directly from the database:
### Open Fauxton
Access the fauxton (CouchDB admin interface) at `<budibase-url>:10000//db/_utils/`

Log in with your COUCHDB_USER and COUCHDB_PASSWORD that can be found in a `.env` file on your server.

### Open Global DB
Find the database called "global-db" and click on it.
<img width="1579" alt="Screenshot 2021-07-27 at 21 48 23" src="https://user-images.githubusercontent.com/11256663/127225205-8798f9ad-1a14-4d83-bcc9-5b4a7409730f.png">

### Delete the admin user
Delete any document starting with `us_`  (users), in the DB.

<img width="1195" alt="Screenshot 2021-07-27 at 21 47 54" src="https://user-images.githubusercontent.com/11256663/127225156-49d55e0f-6f18-4091-a9b4-fcb4ad09856d.png">

Open budibase at `<budibase-url>:10000//db/_utils/` and you should be prompted to create your admin user again.

Hope this helps!

**Reply by CharlesBeatty** [reply]
Thank you. I am tunneling into a remote server and it would have been a real pain to set up remote access to Fauxton in a Docker container. As it happens I spent the last couple of hours making my way into the Couchdb database, which I have never touched before, and then deleting my admin login. After that I was able to set it up again. And then I read your response!

**Reply by shogunpurple** [reply]
Glad you got it resolved!

**Reply by max9933** [reply]
Hi @shogunpurple, I've just come across this thread, I've managed to access CouchDB and delete the BudiBase use account but it is not seeming to allow me to create a new user when going to //db/_utils. It just prompts me to sign back in.
Failing that, when you say clearing the containers, how would I do this? 
Thank you

**Reply by deep410** [reply]
@shogunpurple  : i have tried both the approaches and none of them work and there is no way I am able to proceed now, please assist

**Reply by max9933** [reply]
I just had to wait a while for the DB change to be noticed, it them prompted for a new admin password @deep410 try the GUI in a an hour or so

**Comment by kmille** [comment]
Any update on this? In the curent version, there is no `global-db` table. There is a `_users`, but the content appears a bit weird.
EDIT: args, the view with all the databases is paginated ...
